install-dev

```diff
@@ -0,0 +1,53 @@
+: ${GITHUB_USERNAME:=hortonworks}
+: ${GITHUB_PROJECT:=cloudbreak-deployer}
+ 
+circleciapi="https://circleci.com/api/v1.1"
+circle() {
+    declare path=$1
+    shift
+    
+    local url="$circleciapi/$path"
+    curl -s -G \
+        -H "Accept: application/json" \
+        "$url" "$@"
+}
+
+main() {
+    declare branch=${1:-master}
+
+    local os=$(uname -s | tr "[A-Z]" "[a-z]")
+    local latestartifacts="project/github/$GITHUB_USERNAME/$GITHUB_PROJECT/latest/artifacts?branch=$branch&filter=successful"
+    local artifacts=$(circle $latestartifacts | tr -d '[:space:]')
+    local message=$(echo $artifacts | jq -c ".message" 2>/dev/null)
+   
+    if [ "$?" -gt 0 ]; then 
+        echo "No artifact found for branch: 'master' at https://circleci.com/gh/$GITHUB_USERNAME/$GITHUB_PROJECT"
+        exit 1
+    elif [ "${message}" != "" ]; then
+        echo "CircleCI API returned with message: $message"
+        echo "API call: $circleciapi/$latestartifacts"
+        exit 1
+    fi
+
+    local url=$(echo $artifacts | jq -c ".[].url" | grep $os | tr -d '"')
+    if [ "$?" -gt 0 ]; then 
+        echo "No artifacts found for os: $os at https://circleci.com/gh/$GITHUB_USERNAME/$GITHUB_PROJECT"
+        exit 1
+    fi
+
+    local dest=/bin
```

Comment:
/bin contains "system" binaries, so /usr/bin would be better and match more to POSIX standards. "/bin Essential command binaries that need to be available in single user mode; for all users, e.g., cat, ls, cp."